src/NServiceBus.Metrics/RawData/RingBuffer.cs

```diff
@@ -16,17 +16,19 @@ class RingBuffer
 
         public struct Entry
         {
-            public Entry(long ticks, long value)
+            public Entry(long ticks, long value, int tag = 0)
```

Comment:
I really like the fact that we start with support for only one tag. KISS for the way :).